Makefile.am

```diff
@@ -5,7 +5,7 @@ EXTRA_DIST = *.py *.md LICENSE LICENSE-ATI LICENSE-MIT \
 	nginx_example.conf imgList.xml autogen.sh \
 	armoryengine/*.py ui/*.py \
 	img/* lang/* dpkgfiles/* \
-	windowsbuild/* linuxbuild/* osxbuild/*
+	windowsbuild/* linuxbuild/* osxbuild/* BitcoinArmory.spec
```

Comment:
Does the spec need to be distributed as a standalone file when packaged under Fedora?